### DISCUSSION TITLE
How do use `clear_prefix` with transparent.nvim?

### DISCUSSION THREAD

**Discussion by careb0t** [nix-community/nixvim#2532] [Q&A]
![image](https://github.com/user-attachments/assets/51263298-9378-46aa-821f-4dc99bb8b800)

I am trying to make my lualine transparent, but it has a strange interaction with transparent.nvim that requires using this `clear_prefix` helper function, but there is no nixvim option for transparent.nvim to enable this, so is there any other way I could use this? I haven't ever used the `luaConfig` option for any plugins before, so I don't even really know what its capabilities are, but could this be an option? If so, could anyone show me an example of how one uses `luaConfig`? Or just any other solution for getting this helper function working?

**Comment by khaneliman** [comment]
Yeah, sounds like you could use `luaConfig.post` to set some a raw lua string to call after your plugin loads to clear it.

```nix
plugins.transparent.luaConfig.post = ''
  require('transparent').clear_prefix('lualine')
'';
